php_grpc/worker.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Sample GRPC PHP server.
+ */
+
+use Spiral\Goridge;
+use Spiral\RoadRunner;
+
+ini_set('display_errors', 'stderr');
+require "vendor/autoload.php";
+
+$server = new \Spiral\GRPC\Server();
+$server->registerService(\Helloworld\GreeterInterface::class, new HelloworldService());
+
+$w = new RoadRunner\Worker(new Goridge\StreamRelay(STDIN, STDOUT));
+$server->serve($w);
```

Comment:
the backslashes are ugly, but the implementation is still better than in `C+`. ;)

Reply:
true story